docs/proposal-link-definition-and-disambiguation.md

```diff
@@ -0,0 +1,51 @@
+# Proposal: Link Definition and Disambiguation
+
+## Current status
+
+- We generate link ref defs from wiki-links inline
+- Wiki links are expected to be the full source of truth, but we don't have enough information to conclusively disambiguate
+
+## Solution
+
+- Don't wipe out link ref defs. Only add/remove as needed. This can be our metadata for disambiguation.
+
+Let's say we have two documents in the workspace with the same title "Some Title"
+
+- We would have autocompletion base on title of the document
```

Comment:
could autocomplete always be done with a combination of title and file path within the workspace - in a style of "My title - foo/bar/my-title.md"